src/app/app.container.tsx

```diff
@@ -1,17 +1,22 @@
 import React, { Component } from 'react';
 import { Route, Switch } from 'react-router-dom';
 import { CounterContainer } from '../containers/counter/counter.container';
+import { DetailContainer } from '../containers/detail/detail.container';
 import { HomepageContainer } from '../containers/homepage/homepage.container';
+import { AdminLayoutContainer } from '../containers/sharedlayouts/admin-layout.container';
 import './app.container.less';
 
 export class AppContainer extends Component {
     render() {
         return (
-            <Switch>
-                <Route exact path="/" component={HomepageContainer} />
-                <Route path="/counter" component={CounterContainer} />
-                <Route render={() => <div>No routes here</div>} />
-            </Switch>
+            <AdminLayoutContainer>
+                <Switch>
+                    <Route exact path="/" component={HomepageContainer} />
+                    <Route exact path="/detail" component={DetailContainer} />
```

Comment:
Proposition for routing:
`/localsurvey/survey/690854098d` <- detail page for given survey
`/localsurvey/survey/` <- all surveys
`/localsurvey/survey/690854098d/builder` <- survey builder for given survey

Reply:
I prefer switching this around, for clearer subrouting:
```
/localsurvey/builder
/localsurvey/builder/:id
/localsurvey/surveys
/localsurvey/surveys/:id
```

Reply:
How does `localsurvey/surveys` differ from the root directory `/localsurvey`?

If not at all, perhaps we should land at surveys directly?

Reply:
I agree, at the moment the homepage and survey list are the same, and neither page is likely to grow larger enough to warrant it's own page. We could however redirect root to `/surveys/` for clarity.

Reply:
I was thinking that. Also, what does the homepage really need for content besides that list. Is there even a concept such as "homepage" needed?